runbook.gemspec

```diff
@@ -42,7 +43,7 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "appraisal", "~> 2.2"
   spec.add_development_dependency "aruba", "~> 0.14"
-  spec.add_development_dependency "bundler", "~> 1.15"
+  spec.add_development_dependency "bundler", ">= 2.2.0", "< 3.0"
```

Comment:
why not use `~> 2.2` here, instead?

Reply:
Yeah, that makes sense to me. I can't recall why I expressed it this way. Will update.